### PR TITLE
Add privacy policy

### PR DESCRIPTION
**Pull request by Ghabry**
Lots of boring crap.

Website part is missing. Rest must be refined...

### CODE REVIEW

File: content/privacy/index.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+---
+title: "Privacy Policy"
+no_menu: true
+---
+<div class="info" markdown="1">
+
+## Our website
+
+our IP address is stored for anti-abuse purposes and is not used in any other ways. We do not use any analytics service.
```
Comment (carstene1ns) [inline content/privacy/index.md:9]: *Your

File: content/privacy/index.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+## Our software (Player, Editor, Tools, liblcf and OpenRTP)
+
+Our software does not access the internet. All data is stored locally. When reporting bugs we will ask you to submit some of this data manually.
+
+When you report a bug we ask you to submit the log file (``easyrpg_log.txt``) and the savegames (``SaveXX.lsd``). The log file contains information about the execution of the engine and does not contain any personal data. The savegames do not normally contain any personal data. Some games will ask you for personal data such as your name. This data is then available in the savegame.
+
+When EasyRPG Player crashes under Windows, a memory dump (``dmp``) is created. This can contain personal data such as your Windows user name.
+
+The typical way to report a bug is through GitHub. By using GitHub you agree to the [GitHub Privacy Policy](https://docs.github.com/site-policy/privacy-policies/github-privacy-statemen). Alternatively, you can send us an email. If you use the Android Bug Report function, the log file and your savegames will be attached automatically.
```
Comment (carstene1ns) [inline content/privacy/index.md:37]: */github-privacy-statement

File: content/privacy/index.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+- [Discord Privacy Policy](https://discord.com/privacy)
+- [Libera Chat Privacy Policy](https://libera.chat/privacy)
+- [Matrix Privacy Policy](https://matrix.org/legal/privacy-notice)
+
+### Social Network
+
+We use external service providers to keep you up to date. By visiting these sites you agree to the terms and conditions:
+
+- [Facebook](https://facebook.com/privacy/policy)
+- [Instagram](https://help.instagram.com/155833707900388)
```
Comment (carstene1ns) [inline content/privacy/index.md:58]: Does this have no better link?
it references https://privacycenter.instagram.com/policy at least

**Comment by Ghabry** [comment]
Finished writing the policy.

The following sections were taken from other websites (all seem to steal from each other so is hard to tell which the real source is xD)

- _Third party links_ based on Discourse policy but you find this wording everywhere
- _Information disclosure to outside parties_ based on Discourse policy but you find this wording everywhere
- _Children’s Online Privacy Protection Act Compliance_ based on Discourse policy but you find this wording everywhere

**Review by fdelapena — APPROVED** [review]

**Pull request merged**